src/Makefile.global.in

```diff
@@ -598,6 +598,13 @@ TAS         = @TAS@
 
 ##########################################################################
 #
+# Link to ZMQ
+
+LIBS := -lzmq $(LIBS)
```

Comment:
https://gist.github.com/usmanm/32a54a6b0f1f29d7737f86e29f837afa

This builds a static library as well.